### ISSUE TITLE
ability to group repositories in virtual folders

### ISSUE THREAD

**Issue by toStars21** [sublimehq/sublime_merge#1541]
**Problem description**

when i have a lot of repositories, it is hard to search among them

**Preferred solution**

it would be very convenient to have ability to group repositories in virtual folders/groups

**Comment by tukanos** [comment]
Yes, I just started to write about the same functionality.  It would also be great if you could name and color the groups!

For example,  I have 5 repositories that are needed to be in one group.  In a second group I would have libraries (20+) that would depend on the first 5 repositories.  

I now have separated windows, but it is really annoying when I accidentally close one window.
